template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Views/IdentityService/LoggedOut.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    ViewData["Title"] = "Sign Out";
```

Comment:
Sign out? Log out? Do we want to be more consistent here? Or is it inconsistent because OIDC/B2C/ASP.NET already disagree on the terminology?